app/src/main/java/ar/scacchipa/jsonplaceholderviewer/data/ITypicodeSourceData.kt

```diff
@@ -0,0 +1,27 @@
+package ar.scacchipa.jsonplaceholderviewer.data
+
+import com.google.gson.GsonBuilder
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface ITypicodeSourceData {
+    @GET("posts")
+    suspend fun getPosts(): Response<List<Post>>
+
```

Comment:
now we have 2 repositories, one for each data type , we shoudl also slipt this datasource in two

Reply:
ITypicodeSourceData was splited into IRemotePostSourceData and IRemotePostSourceData